mimoto-issuers-config.json

```diff
@@ -151,64 +151,64 @@
       "credential_issuer": "Sunbird",
       "display": [
         {
-          "name": "Veridonia Insurance Company",
+          "name": "StayProtected Insurance",
           "logo": {
-            "url": "https://${mosip.api.public.host}/inji/veridonia-logo.png",
-            "alt_text": "a Veridonia logo"
+            "url": "https://raw.githubusercontent.com/tw-mosip/file-server/master/StayProtectedInsurance.png",
+            "alt_text": "a square logo of a StayProtected"
           },
           "language": "en",
-          "title": "Veridonia Insurance Company",
-          "description": "Download Insurance Credentials using policy details"
+          "title": "Download StayProtected Insurance Credentials",
+          "description": "Download insurance credential"
```

Comment:
title and description looking similar, Download credential
should we change title to `StayProtected Insurance` or `StayProtected Insurance Department`

Reply:
Modified to "StayProtected Insurance" in title which will be the title in the mobile, similar to name which will be title in the web.